### PR TITLE
patch release 1.4.2

### PR DESCRIPTION
**Pull request by ehdsouza**

### CODE REVIEW

**Comment by ehdsouza** [comment]
@mediumTaj thanks! added in commit: https://github.com/watson-developer-cloud/python-sdk/pull/596/commits/09689c644e9bbd805a695bfe3498e2bc0466b205

File: watson_developer_cloud/speech_to_text_v1_adapter.py [hunk 1]
```diff
@@ -141,6 +145,10 @@ def recognize_using_websocket(self,
         labels](https://console.bluemix.net/docs/services/speech-to-text/output.html#speaker_labels).
         :param str http_proxy_host: http proxy host name.
         :param str http_proxy_port: http proxy port. If not set, set to 80.
+        :param str customization_id: **Deprecated.** Use the `language_customization_id`
+        parameter to specify the customization ID (GUID) of a custom language model that
+        is to be used with the recognition request. Do not specify both parameters with a
+        request.
```
Comment (mediumTaj) [inline watson_developer_cloud/speech_to_text_v1_adapter.py]: 👍

File: watson_developer_cloud/speech_to_text_v1_adapter.py [hunk 2]
```diff
@@ -178,7 +186,8 @@ def recognize_using_websocket(self,
             'customization_id': customization_id,
             'acoustic_customization_id': acoustic_customization_id,
             'customization_weight': customization_weight,
-            'base_model_version': base_model_version
+            'base_model_version': base_model_version,
+            'language_customization_id': language_customization_id,
```
Comment (mediumTaj) [inline watson_developer_cloud/speech_to_text_v1_adapter.py]: Does the order of params matter here? do we need to switch the positions of `customization_id` and `language_customization_id`?

Comment (ehdsouza) [reply watson_developer_cloud/speech_to_text_v1_adapter.py]: The order doesnt matter here.

**Pull request merged**